resources/assets/js/components/PresetSelection.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+    <div>
+        <h3>Voeg voorgedefineerde momenten toe</h3>
```

Comment:
Labels & text should be translatable?

Reply:
No.
There's talk of making user input multilingual one day though.